lemur/notifications/messaging.py

```diff
@@ -96,11 +97,20 @@ def send_notification(event_type, data, targets, notification):
     :param notification:
     :return:
     """
+    function = f"{__name__}.{sys._getframe().f_code.co_name}"
+    log_data = {
+        "function": function,
+        "message": "Sending expiration notification for to targets {}".format(targets),
+        "notification_type": "expiration",
+        "certificate_targets": targets,
+    }
     status = FAILURE_METRIC_STATUS
     try:
         notification.plugin.send(event_type, data, targets, notification.options)
         status = SUCCESS_METRIC_STATUS
     except Exception as e:
+        log_data["message"] = "Unable to send expiration notification to targets {}".format(targets)
+        current_app.logger.error(log_data, exc_info=True)
```

Comment:
this might be something worthwhile alerting, and adding metrics would facilitate that, similar to:
https://github.com/Netflix/lemur/blob/master/lemur/common/celery.py#L552-L553

Reply:
For the record: I spoke to Hossein, and we've confirmed there is already metric info for failed notifications a few lines below this, so no changes needed.